Adding the ADR for using English language as default for coding (#131)
MinBZK/ai-validation: docs/ADRs/0014-written-language.md

# ADR-0014 Written Language

## Context

In order to expand our reach and foster international collaboration in the field of AI Validation, we have decided to
conduct all communication in English on public platforms such as GitHub. This decision aims to facilitate better

> understanding and participation from our global colleagues. However, within the Dutch Government, the norm is to

Reviewer: Official translation of Rijksoverheid in English is Government of the Netherlands. 

```suggestion
understanding and participation from our global colleagues. However, within the Government of the Netherlands, the norm is to
```

> communicate in Dutch for internal purposes. This ADR will provide  guidelines on which language to use for different

Reviewer: ```suggestion
communicate in Dutch for internal purposes. This ADR will provide guidelines on which language to use for different
```

types of communications.

## Assumptions

> There is no requirement to use Dutch as the primary language for all our activities while working for the Dutch
> government. More information can be found in the [More Information](#more-information) section.

Reviewer: ```suggestion
There is no requirement to use Dutch as the primary language for all our activities while working for the Government of the Netherlands. More information can be found in the [More Information](#more-information) section.
```

## Decision

The following channels will utilize English:

> - GitHub Projects
> - GitHub Repository
> - Email to International partners

Reviewer: ```suggestion
- GitHub projects
- GitHub repository
- Email to international partners
```

The primary language for the following channels will be Dutch:

> - Mattermost (Internal Collaboration tool)
> - Email to Internal parties
> - Official Internal Documents like memo's or notes to the chamber
> - Guides for Dutch Users on how to use the tools
> - UI for the tools we will make

Reviewer: Tweede Kamer in English is House of Representatives: https://en.wikipedia.org/wiki/House_of_Representatives_(Netherlands)

```suggestion
- Mattermost (internal collaboration tool)
- Email to internal parties
- Official Internal documents like memo's or notes to the house of representatives
- Guides for Dutch users on how to use the tools
- UI for the tools we will make
```

## Risks

Dutch-only developers will have a harder time following along with the progression of our team on both the code on
GitHub as our Project Management.

## Consequences

> - All tickets and issues will be written in English on GitHub projects
> - The review on code will be written in English
> - Comments in the code and commit messages will be written in English
> - Guides on usage of the tools will be written in both Dutch and English
> 

Reviewer: ```suggestion
- All tickets and issues will be written in English on GitHub projects.
- Code reviews will be written in English.
- Comments in the code and commit messages will be written in English.
- Documentation of the tools will be written in both Dutch and English.

```

## More Information

> Although many attempts by the previous cabinets, Dutch is not an official language in the Netherlands according to the

Reviewer: ```suggestion
Although many attempts by the previous cabinets, Dutch is not the official language in the Netherlands according to the
```

Dutch constitution see the following
[link](https://www.denederlandsegrondwet.nl/id/viivckl8ibxx/nederlandse_taal_in_de_grondwet).

> According to the [website of the central government](https://www.rijksoverheid.nl/onderwerpen/erkende-talen/vraag-en-antwoord/erkende-talen-nederland)

Reviewer: ```suggestion
According to the [website of the Government of the Netherlands](https://www.rijksoverheid.nl/onderwerpen/erkende-talen/vraag-en-antwoord/erkende-talen-nederland)
```

> the Dutch language is the official recognized language. Meaning in combination with the law `Algemene wet bestuursrecht`
> on [wetten.overheid.nl](https://wetten.overheid.nl/BWBR0005537/2024-01-01) that governing bodies and their employees
> needs to be communicating in Dutch unless stated differently elsewhere. And especially is stated
> [here](https://wetten.overheid.nl/jci1.3:c:BWBR0005537&hoofdstuk=2&afdeling=2.2&artikel=2:6&z=2024-01-01&g=2024-01-01)
> that can be deviated from it unless there is more efficient goal.

Reviewer: ```suggestion
the Dutch language is the official recognized language. This means that in combination with the law `Algemene wet bestuursrecht`
on [wetten.overheid.nl](https://wetten.overheid.nl/BWBR0005537/2024-01-01) governing bodies and their employees
need to communicate in Dutch unless stated differently elsewhere. It is stated
[here](https://wetten.overheid.nl/jci1.3:c:BWBR0005537&hoofdstuk=2&afdeling=2.2&artikel=2:6&z=2024-01-01&g=2024-01-01) that communicating in another language than Dutch is permitted if the goal of communicating in another language than Dutch is sufficiently justified and if other parties are not effected disproportionately by the usage of another language. 
```

# ADR-0014 Written Language

## Context

In order to expand our reach and foster international collaboration in the field of AI Validation, we have decided to
conduct all communication in English on public platforms such as GitHub. This decision aims to facilitate better
understanding and participation from our global colleagues. However, within the Government of the Netherlands, the
norm is to communicate in Dutch for internal purposes. This ADR will provide guidelines on which language to use for
different types of communications.

## Assumptions

There is no requirement to use Dutch as the primary language for all our activities while working for the Government of
the Netherlands. More information can be found in the [More Information](#more-information) section.

## Decision

The following channels will utilize English:

- GitHub projects
- GitHub repository
- Email to international partners

The primary language for the following channels will be Dutch:

- Mattermost (internal collaboration tool)

> - Email to internal parties

Reviewer: ```suggestion
- Emails to internal parties
```

> - Official Internal documents like memo's or notes to the house of representatives

Reviewer: ```suggestion
- Official internal documents like memo's or notes to the house of representatives
```

- Guides for Dutch users on how to use the tools
- UI for the tools we will make

## Risks

Dutch-only developers will have a harder time following along with the progression of our team on both the code on
GitHub as our Project Management.

## Consequences

- All tickets and issues will be written in English on GitHub projects.
- Code reviews will be written in English.
- Comments in the code and commit messages will be written in English.
- Documentation of the tools will be written in both Dutch and English.

## More Information

> Although many attempts by the previous cabinets, Dutch is not the official language in the Netherlands according to the

Reviewer: ```suggestion
Although many attempts by previous cabinets, Dutch is not the official language in the Netherlands according to the
```

> Dutch constitution see the following

Reviewer: ```suggestion
Dutch constitution. See the following
```
